files/class-api-client.php

```diff
@@ -37,10 +37,10 @@ public function __construct( $api_base, $files_site_id, $files_token, $cache ) {
 		$this->files_token   = $files_token;
 
 		// Add some context to the UA to simplify debugging issues
-		if ( defined( 'DOING_CRON' ) && DOING_CRON ) {
+		if ( defined( 'DOING_CRON' ) && constant( 'DOING_CRON' ) ) {
 			// current_filter may not be totally accurate but still better than nothing
 			$current_context = sprintf( 'Cron (%s)', current_filter() );
-		} elseif ( defined( 'WP_CLI' ) && WP_CLI ) {
+		} elseif ( defined( 'WP_CLI' ) && constant( 'WP_CLI' ) ) {
```

Comment:
With constant mocking, whenever we use `defined()`, we have to use `constant()`. Otherwise, when the constant was defined in the locker and then it is later accessed without `constant()`, PHP will spit an error because it knows nothing about the constant.

In PHP 8.1, undefined constants trigger fatal errors.